docs/docsite/rst/administration/instances.rst

```diff
@@ -51,8 +71,36 @@ Prerequisites
 - To manage instances from the AWX user interface, you must have System Administrator or System Auditor permissions.
 
 
+Using a custom Receptor CA
```

Comment:
Might be better to move this section lower, since it is optional. Maybe before or after the troubleshooting section